package/MDAnalysis/analysis/density.py

```diff
@@ -384,6 +391,9 @@ class DensityAnalysis(AnalysisBase):
     .. versionadded:: 1.0.0
     .. versionchanged:: 2.0.0
        :func:`_set_user_grid` is now a method of :class:`DensityAnalysis`.
+    .. deprecated:: 2.0.0
```

Comment:
Move to the attribute?

Reply:
Thanks for the review. Done, I'll do the same for the other two PRs.